addons/website/static/src/js/menu/edit.js

```diff
@@ -33,6 +33,8 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         edition_was_stopped: '_onEditionWasStopped',
         request_save: '_onSnippetRequestSave',
         request_cancel: '_onSnippetRequestCancel',
+        suspend_edit_observer: '_onSuspendEditObserver',
+        restart_edit_observer: '_onRestartEditObserver',
```

Comment:
This commit is suspicious. I think the editor is already fixing issues like those, did you check with them to make this commit? If not, do this: create a dedicated PR for this commit only and send it to them to check if they are not already fixing the problem another way (I know they are aware of the o_dirty flag issue everywhere).